### DISCUSSION TITLE
[Question]: I attached a excel file in chat that it's very slow. The excel file had 300 rows and 20 columns and text is too long.

### DISCUSSION THREAD

**Discussion by angpao** [danny-avila/LibreChat#4246] [Q&A]
### What is your question?

How to increase speed ?

### More Details

How to increase speed ?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by angpao** [comment]
I installed Librechat on vm 1 cpu  ram 3.75 and used deploy-compose.yaml.

**Comment by danny-avila** [comment]
This is expected behavior. CSV/Excel files can be "text" heavy despite relatively small sizes. These filetypes are not optimal for "RAG". There is an alternative strategy required when dealing with these file types that is not yet implemented. Track the open issue on the rag api repo for further discussion and staying up to date with this feature:

https://github.com/danny-avila/rag_api/issues/54
